src/ucp/core/ucp_context.c

```diff
@@ -2114,6 +2147,9 @@ static ucs_status_t ucp_fill_config(ucp_context_h context,
              ((context->config.ext.max_rma_lanes > 1) ||
               context->config.ext.proto_enable));
 
+    context->config.progress_wrapper_enabled =
+            ucs_log_is_enabled(UCS_LOG_LEVEL_TRACE_REQ) ||
+            ucp_context_usage_tracker_enabled(context);
```

Comment:
This config field causes short proto disabling short if `UCS_LOG_LEVEL_TRACE_REQ` log level is enabled. I don't think that make proto selection dependent on log level is a good idea.

E.g. for protov1 setting `UCX_LOG_LEVEL=req` is one of the simplest ways to understand what procol was chosen for MPI calls. 

Can we avoid that somehow?